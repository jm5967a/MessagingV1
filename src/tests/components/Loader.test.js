import React from 'react';
import {shallow} from 'enzyme';
import Loader from '../../components/LoadingPage'

test("Should render Loader", ()=> {
   const wrapper = shallow(<Loader/>);
   expect(wrapper).toMatchSnapshot();
});