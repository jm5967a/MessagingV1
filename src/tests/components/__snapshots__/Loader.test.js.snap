// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Loader 1`] = `
<div
  className="loader"
>
  <img
    className="loader__image"
    src="/images/loader.gif"
  />
</div>
`;
